packages/manager/modules/pci/src/components/project/flavor-billing/flavor-billing.controller.js

```diff
@@ -30,10 +35,35 @@ export default class FlavorBillingController {
   }
 
   getPrice(price) {
-    return this.number * price.value;
+    return this.number * (price.value || price);
   }
 
   formatPrice(price) {
     return this.PriceFormatter.format(this.getPrice(price));
   }
+
+  formatAddonsPrice = (price) => price / 100000000;
+
+  getTotalPrice() {
```

Comment:
**nitpick:** Function name is misleading, the function just returns the add-on price, but the name is `getTotalPrice`.
```suggestion
  getAddOnPrice() {
```